index.js

```diff
@@ -151,30 +151,53 @@ module.exports = {
       },
     ],
     'react/self-closing-comp': 'error',
-    "strict": 0,
-    "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "warn",
-    "no-unused-vars": "error",
-    "no-multiple-empty-lines": [
-      "error",
+    'strict': 0,
+    'react-hooks/rules-of-hooks': 'error',
+    'react-hooks/exhaustive-deps': 'warn',
+    'no-unused-vars': 'error',
+    'no-multiple-empty-lines': [
+      'error',
       {
-        "max": 1,
-        "maxEOF": 1
+        'max': 1,
+        'maxEOF': 1
       }
     ],
-    "space-unary-ops": "error",
-    "jest/no-disabled-tests": "warn",
-    "jest/no-focused-tests": "error",
-    "jest/prefer-to-have-length": "warn",
-    "react/react-in-jsx-scope": "off",
-    "jest/valid-expect": "off",
-    "no-trailing-spaces": "off",
-    "jest/no-identical-title": "off",
-    "no-shadow": "off",
-    "eqeqeq": "off",
-    "consistent-return": "off",
+    'space-unary-ops': 'error',
+    'jest/no-disabled-tests': 'warn',
+    'jest/no-focused-tests': 'error',
+    'jest/prefer-to-have-length': 'warn',
+    'react/react-in-jsx-scope': 'off',
+    'jest/valid-expect': 'off',
+    'no-trailing-spaces': 'off',
+    'jest/no-identical-title': 'off',
+    'no-shadow': 'off',
+    'eqeqeq': 'off',
+    'consistent-return': 'off',
     'no-unused-expressions': 'off',
     'import/namespace': ['error', { allowComputed: true }],
     'import/no-unresolved': 'warn'
   },
+  'simple-import-sort/imports': [
+    'error',
+    {
+      groups: [
+        // Packages `react` related packages come first.
+        ['^react', '^[a-z]'],
+        // Packages from a '@deriv' scope come second.
+        ['^@\\w'],
+        // Absolute imports and other imports from aliases like 'Components/...'
+        ['^[A-Z]'],
+        // Internal packages.
+        ['^(@|components)(/.*|$)'],
+        // Side effect imports.
+        ['^\\u0000'],
```

Comment:
@mahdiyeh-deriv Can you explain the use of this regex? `^\\u0000`

Reply:
This represents the imports that start with the null character. However, it's relatively unusual to encounter import statements that begin with the null character in regular codebases so I can remove it from here.